src/controllers/insertIssueTable.gs

```diff
@@ -0,0 +1,268 @@
+    if (true === this.hasOverlappingTable(true) ) {
+      // will show warning/alter and skip opening the dialog
+      return;
+    }
+
+    var customFields = IssueFields.getAvailableCustomFields(IssueFields.CUSTOMFIELD_FORMAT_SEARCH);
+    var userColumns = UserStorage.getValue('userColumns') || [];
+    var dialog = getDialog('views/dialogs/insertIssueFromFilter', {
+      columns : IssueFields.getBuiltInJiraFields(),
+      customFields : customFields,
+      userColumns : userColumns.length > 0 ? userColumns : jiraColumnDefault
+    });
+
+    // try to adjust height depending on amount of jira fields to show
+    var rowH = 32;
+    var height = 424;
+    height += (Math.ceil(Object.keys(IssueFields.getBuiltInJiraFields()).length % 4) * rowH);
+    height += (Math.ceil(Object.keys(customFields).length % 4) * rowH);
+
+    dialog
+      .setWidth(600)
+      .setHeight(height)
+      .setSandboxMode(HtmlService.SandboxMode.IFRAME);
+
+    debug.log('Processed: %s', dialog);
+
+    SpreadsheetApp.getUi().showModalDialog(dialog, 'List Jira issues from filter');
+  },
+
+  /**
+   * @desc Form handler for dialogIssuesFromFilter. Retrieve issues for given filter with specified columns from Jira and insert into
+   *       current active sheet.
+   * @param jsonFormData {object} JSON Form object of all form values
+   * @return {object} Object({status: [boolean], response: [string]})
+   */
+  callbackInsertFromFilter : function (jsonFormData) {
+    debug.log(this.name + '.callbackInsertFromFilter() <= %s', JSON.stringify(jsonFormData));
+
+    jsonFormData = jsonFormData || {filter_id: 0};
+    var that = this,
+        startAt = parseInt(jsonFormData['startAt']) || 0,
+        columns = jsonFormData['columns'] || jiraColumnDefault,
+        response = {status: false, message: ''};
+
+    var Renderer, attributes = {
+      filter : jsonFormData['filter_id'] ? getFilter(parseInt(jsonFormData['filter_id'])) : {},
+      maxResults : parseInt(jsonFormData['maxResults']) || 10000,
+      issues : {},
+      columns : columns,
+      sheet : getTicketSheet(),
+      renderer : IssueTableRendererDefault_
+    };
+
+    UserStorage.setValue('userColumns', columns); // store for re-use by user
+
+    var ok = function (resp, status, errorMessage) {
+      debug.log(this.name + '.ok() resp(len): %s; s: %s; msg: %s', resp.data.length, status, errorMessage);
+
+      if (status !== 200) {
+        // Something funky is up with the JSON response.
+        response.message = "Failed to retrieve jira issues!";
+        Browser.msgBox(response.message, Browser.Buttons.OK);
+      } else if (resp.data.length === 0) {
+        // any issues in result?
+        response.message = "No issues were found to match your search.";
+        Browser.msgBox(response.message, Browser.Buttons.OK);
+
+        return;
+      } else {
+        attributes.issues = resp.data;
+
+        var Table = new IssueTable_(attributes);
+        if (Renderer = Table.render()) {
```

Comment:
Its not clear what this condition is doing here.
I *think* Render is the return value from Table.render() and you are checking for a non- null value
better done like this imho for readability
`var Renderer = Table.render();
if (Renderer != null)`
also not sure why you are using a capital letter at the start of the variable - that's for class names I think?

Reply:
For my knowledge that a "improved" short version of what you wrote in 2 lines.
I do a !null or ! false check at same time of return value assignment to _Renderer_ .
As "Renderer" return value is indeed a object, i thought a capital letter would be appropiate here.

Reply:
The widely used version of the standard is for capital letters are used for the class definition/name. This is an instance of a class (which pretty much all variables are) so by following that standard should be lower case.
I still find the code hard to read - not a release blocker.